deployment/gcp/predict.py

```
@@ -1,149 +1,177 @@
-            return jsonify({'error': 'Empty text provided'}), 400
-        
+            return jsonify({"error": "Empty text provided"}), 400
+
         # Make prediction
         result = model.predict(text)
-        
+
         return jsonify(result)
-        
+
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)